pyvista/plotting/composite_mapper.py

```diff
@@ -341,13 +351,18 @@ def reset_pickabilities(self):
         to our indexing to access the right block.
 
         >>> import pyvista as pv
-        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> dataset = pv.MultiBlock(
+        ...     [pv.Cube(), pv.Sphere(center=(0, 0, 1))]
+        ... )
         >>> pl = pv.Plotter()
         >>> actor, mapper = pl.add_composite(dataset)
         >>> mapper.block_attr[1].pickable = True
         >>> mapper.block_attr[2].pickable = False
         >>> mapper.block_attr.reset_pickabilities()
-        >>> [mapper.block_attr[1].pickable, mapper.block_attr[2].pickable]
+        >>> [
```

Comment:
Consider making two separate commands, but this is a weak suggestion from me.

Reply:
Fine as is IMO.